maintainers/maintainer-list.nix

```diff
@@ -2613,6 +2613,12 @@
     githubId = 231483;
     name = "Jack Kelly";
   };
+  enolan = {
+    name = "Echo Nolan";
+    email = "echo@echonolan.net";
```

Comment:
Please put this in a separate commit in the form of ``maintainers: add name``.

Reply:
done